### ISSUE TITLE
Bug report usin Win Shift D. Screenshot 

### ISSUE THREAD

**Issue by IIEleven11** [cairoshell/cairoshell#458]
**Describe the bug**
I was testing functionality/ to see what it did. Recieved pop up error. Screenshot will be attached
Shell was then closed leaving me without any taskbar/menu. Went into task manager and started explorer from there.
**Steps to reproduce**
1. Win+shift+D
2. 
3. 

**Expected behavior**
Bug report window should pop up. Cairo shelll should close

**System information:**
 - Windows version: 10 home 2004
 - Cairo version: Version 0.3.7108.0 - Pre-release
 - 32-bit or 64-bit: 64
 - In Cairo Settings > Advanced, is Cairo set as shell?: No

**Screenshots**
![image](https://user-images.githubusercontent.com/55998857/96054668-fcf6fd00-0e36-11eb-869e-a81502857059.png)

**Comment by dremin** [comment]
Thanks for the report! A couple questions:
- This key combo triggers the desktop overlay. When the crash happened, were you trying to open or close the desktop overlay?
- Do you have multiple monitors? If so, what is the layout of the monitors?
- Does this happen every time you press this key combo while Cairo is running?

**Comment by dremin** [comment]
@IIEleven11 I fixed a different problem and am wondering if it also fixed this issue. Could you please try [this build](https://ci.appveyor.com/api/buildjobs/wa36ibk587a5kbfw/artifacts/Installer%2FCairoSetup_64bit.exe)? Thanks!

**Comment by IIEleven11** [comment]
sorry, i forgot i posted this. So when this happned I accidentally held it down. I can actually reproduce this at will i just noticed. If I hold down win+ shift + D a little too long. i'll get the same pop up error window. 

I am using multiple monitors. My main screen is 1920x1080 @ 144hz and i have a vertically mounted monitor that is set at 768x1366 (native) @ 60hz. I'm using display fusion to control this correctly as i know this  is an odd setup. It works though. I also get cairo on both screens . 

So I can push win shift D right now with reasonable speed and it will work as intended. But if i hold it down a second too long. i'll get the error and cairo will close.

**Comment by IIEleven11** [comment]
I will try that build now

**Comment by dremin** [comment]
Thanks for the details. Based on what you've said I believe the build I linked should fix the problem, I'd be interested to hear if it does!

**Comment by IIEleven11** [comment]
It looks like you hit the nail on the head and got it. I was even trying to break it and couldnt. It did what was expected, holding it down flashed the overlay on and off repeatedly  until i let go. Thank you!

Hey while i have your attention is there somewhere people are sharing skins or somewhere I can get more info on how to skin this? Heres a screen shot of what I figured out, but is this about the distance of the possibilities? I have limited C++ knowledge which doesnt help me here. I was able to piece together some stuff through what you linked in the wiki/trial and error, but it was pretty time consuming. 

Also is there a way or place I can put the backward/forward/folder/home button. If i keep it on the desktop it just gets overlapped by everything. And if its up on the task bar and i have too much open it will overlap sections. 
![image](https://user-images.githubusercontent.com/55998857/96519542-11b20700-1222-11eb-9d30-21bcb5ac0582.png)
